pkg/sql/parser/sql.y

```diff
@@ -4741,6 +4741,10 @@ logical_replication_options:
   {
     $$.val = &tree.LogicalReplicationOptions{SkipSchemaCheck: tree.MakeDBool(true)} 
   }
+| LABEL '=' string_or_placeholder
```

Comment:
looks like cdc uses `metric_label` https://www.cockroachlabs.com/docs/stable/create-changefeed#options

Might be nice to keep it consistent, but also i wish they just used label as their keyword too.

Reply:
CDC does’t parse their options — they just use string_or_placeholder and then match/validate during query exec — so they don’t reserve keywords. If we reuse their option we would need to reserve it.

Reply:
ah, didn't realize cdc doesn't parse options. carry on.